_posts/release/2020-01-19-accumulo-2.1.0.md

```diff
@@ -277,6 +277,14 @@ along with the addition of a "Fetched" column indicating when the data was retri
 for External Compactions that allows seeing the progress of compactions and many details about ongoing compactions.
 {% ghi 2283 %} {% ghi 2294 %} {% ghi 2358 %}
 
+### New tserver scan timeout property
+
+The new property tserver.max.scan.result.timeout was added to allow configuration of the timeout. A bug was
+discovered where tservers were running out of memory, partially due to this timeout being so short. The default value
+is 1 second but now can be increased. It is the max time for the thrift client handler to wait for scan results before
+timing out. {% ghi 2599 %} {% ghi 2598 %}
```

Comment:
```suggestion
The new property `tserver.max.scan.result.timeout` was added to allow configuration of the timeout. A bug was
discovered where tservers were running out of memory, partially due to this timeout being so short. The default value
is 1 second, but now it can be increased. It is the max time for the thrift client handler to wait for scan results before
timing out. {% ghi 2599 %} {% ghi 2598 %}
```

Reply:
I updated the property name so this will have to change. It is now "tserver.scan.results.max.timeout"